### ISSUE TITLE
Cannot find module 'bytes'

### ISSUE THREAD

**Issue by Mirodil** [senchalabs/connect#612]
```
module.js:340
    throw err;
          ^
Error: Cannot find module 'bytes'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
    at Object.<anonymous> (..\connect\lib\utils.js:395:22)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:362:17)
```

I'm using node version "0.8.0" on windows box.

**Comment by yunhengzhen** [comment]
F:\Project\JS\HiChat>node server.js
module.js:338
    throw err;
    ^

Error: Cannot find module 'bytes'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (F:\Project\JS\HiChat\node_modules\express\node_module
s\connect\lib\utils.js:393:22)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)

**Comment by dougwilson** [comment]
Hi @yunhengzhen , the current version of connect published to npm is 3.4.0, which does not have a `lib\utils.js` or a dependency on the `bytes` module. Please upgrade connect at your earliest convenience.
